runners/pvc_autoresizer.go

```diff
@@ -144,6 +144,12 @@ func (w *pvcAutoresizer) reconcile(ctx context.Context) error {
 				log.Info("failed to get volume stats")
 				continue
 			}
+
+			// To output the metric before the first event, we call Nop() here.
+			metrics.ResizerSuccessResizeTotal.Nop(pvc.Name, pvc.Namespace)
+			metrics.ResizerFailedResizeTotal.Nop(pvc.Name, pvc.Namespace)
+			metrics.ResizerLimitReachedTotal.Nop(pvc.Name, pvc.Namespace)
```

Comment:
When we failed to get volume stats, should these calls be skipped?

Reply:
Updated to output metrics when the PVC is a target.